Firebase/Messaging/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
-# Unreleased
+# 2018-07-31 -- v3.1.0
+- Added function to support grobal data flag
```

Comment:
Consider rephrasing this to "Added support for global Firebase data collection flag." Also, include the PR number at the end of the line, if possible.

Reply:
s/grobal/global/

Reply:
Done